ivy/functional/ivy/general.py

```diff
@@ -237,10 +264,7 @@ def get_referrers_recursive(
             val = this_repr
         ret_cont[str(ref_id)] = val
     return ret_cont
-
-
-@handle_exceptions
-@handle_backend_invalid
+    
```

Comment:
why was the decorators removed?

Reply:
My bad. I must have mistakenly removed those decorators. They have been put back